### DISCUSSION TITLE
Add new fields createdById and updatedById

### DISCUSSION THREAD

**Discussion by FelixMalfait** [twentyhq/twenty#4119] [Ideas]
We already have createdAt and updatedAt on every record.
Salesforce also has createdById and updatedAtById.
The first one seems especially valuable. I think we should add both. 

We could argue that these columns should not be created and that this should be solved exclusively by the upcoming "audit logs" table, but this would come with limitation (e.g. unlikely we could ever achieve something as simple as "filter to show only the companies I created").

**Comment by Bonapara** [comment]
Linked to https://github.com/twentyhq/twenty/issues/5155

**Comment by BOHEUS** [comment]
Both createdBy and updatedBy fields are now available to all objects by default
